zebra-network/src/meta_addr.rs

```diff
@@ -530,6 +531,36 @@ impl MetaAddr {
         is_node && self.address_is_valid_for_outbound()
     }
 
+    /// Should this peer considered reachable?
+    ///
+    /// A peer is considered unreachable if the last connection attempt to it failed and the last
+    /// successful connection is more than 3 days ago.
```

Comment:
Nit: remove negation from the docs as well

I'm not sure if we should mention the `AttemptPending` state here, because it only lasts for a short time (4 seconds), and the `was_connection_recently_attempted` skips it.

```suggestion
    /// A peer is probably reachable if:
    /// - it has never been attempted, or
    /// - the last connection attempt was successful, or
    /// - the last successful connection was less than 3 days ago.
```